pkgs/applications/networking/browsers/w3m/default.nix

```diff
@@ -1,16 +1,18 @@
 { stdenv, fetchurl
-, sslSupport ? true
-, graphicsSupport ? false
-, mouseSupport ? false
-, ncurses, openssl ? null, boehmgc, gettext, zlib
-, imlib2 ? null, xlibsWrapper ? null, fbcon ? null
-, gpm-ncurses ? null
+, ncurses, boehmgc, gettext, zlib
+, sslSupport ? true, openssl ? null
+, graphicsSupport ? true, imlib2 ? null
+, x11Support ? graphicsSupport, libX11 ? null
```

Comment:
`w3m-batch`  is not darwin friendly since `w3m` refuses to evaluate because of `gpm` by having `mouseSupport` set to true by default


hmm I'm not too familiar with nixpkgs, but I'm currently using `callPackage` instead of `override` and I guess the same would be required for `w3m-nox`.
```
w3m-batch = callPackage ../applications/networking/browsers/w3m {
  graphicsSupport = false;
  x11Support = false;
  mouseSupport = false;
};
```